canonical-websites/build.snapcraft.io
# Vanilla notification component

## Done

Updated notification component to use CSS module generated from Vanilla styles.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Sign in
- Wait for Beta notification - it should have vanilla styles


## Issue / Card

Fixes #890 

## Screenshots

<img width="1046" alt="screen shot 2017-07-21 at 15 42 50" src="https://user-images.githubusercontent.com/83575/28466084-72339b82-6e2b-11e7-95e9-f6a25428b5ab.png">
<img width="1047" alt="screen shot 2017-07-21 at 15 42 37" src="https://user-images.githubusercontent.com/83575/28466083-7233611c-6e2b-11e7-98c2-a7fec8d1e314.png">
